modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -3320,6 +3320,54 @@ public void testBuildTemplateWorkspaceLocalProperties() throws Exception {
 			workspaceProjectDir, homeDirName + "/osgi/modules/foo.portlet.jar");
 	}
 
+	@Test
+	public void testBuildTemplateWorkspaceWith70() throws Exception {
+		File gradleWorkspaceProjectDir = _buildTemplateWithGradle(
+			WorkspaceUtil.WORKSPACE, "withportlet", "--liferayVersion", "7.0");
+
+		_testContains(
+			gradleWorkspaceProjectDir, "gradle.properties", true,
+			".*liferay.workspace.bundle.url=.*liferay.com/portal/7.0.*");
+
+		File gradleProperties = new File(
```

Comment:
This is a file, so the variable name should end with `File` or `Dir`

Reply:
Oh I missed that one, thanks